core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/state/ReplicationState.java

```diff
@@ -60,4 +94,36 @@ private LongTokenRange convert(TokenRange range)
         long end = (long) range.getEnd().getValue();
         return new LongTokenRange(start, end);
     }
+
+    private static class KeyspaceReplication
+    {
+        private final ImmutableMap<LongTokenRange, ImmutableSet<Host>> myReplication;
+
+        public KeyspaceReplication(ImmutableMap<LongTokenRange, ImmutableSet<Host>> replication)
```

Comment:
Access modifier here an some other places could be narrowed down in this file.

Reply:
Removed the class